kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/enums/CustomResourceStatus.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.enums;
+
+/**
+ * The enum Custom resource status.
+ */
+public enum CustomResourceStatus {
```

Comment:
These system tests will just interested in knowing whether certmanager, strimzi and eventually kroxylicious resources are ready.  We probably won't be interested in the minutiae of Strimzi's states.   We can probably just simplify the predicate to make it match a string Ready.

Reply:
Removed the enum and checked always agains "Ready" string.